SpatialGDKEditorToolbar/Source/Private/SchemaGenerator/Utils/DataTypeUtilities.h

```diff
@@ -32,3 +32,8 @@ FString SchemaRPCName(UClass* Class, UFunction* Function);
 
 // Given a property node, generates the schema field name.
 FString SchemaFieldName(const TSharedPtr<FUnrealProperty> Property);
+
+FString AlphaNumericSanitization(const FString& InString);
+
+// Regex pattern matcher to match alphanumeric characters.
+const FRegexPattern AlphaNumericPatern(TEXT("[A-Z,a-z,0-9]"));
```

Comment:
I believe this will give a duplicate symbol error if you're not using unity build.